force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -176,7 +203,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The average of the three numbers.
      */
     public static Double averageOfThreeNumbers(Integer a, Integer b, Integer c) {
-        return null; // Replace null with the variable you used to store the result
+        Double avgValue = null;
+        avgValue = ((Double)a + (Double)b + (Double)c) / 3;
+        return avgValue; // Replace null with the variable you used to store the result
```

Comment:
Logically correct but could be more efficient.  No need to assign null to the value and then on the next long the results of the math. 

	return (a + b + c) / 3.0;